assets/stories/skate-components/detours/diversionPanel.stories.tsx

```diff
@@ -14,6 +14,7 @@ const meta = {
     routeDescription: "Harvard via Allston",
     routeOrigin: "from Andrew Station",
     routeDirection: "Outbound",
+    onChangeRoute: undefined,
```

Comment:
Honestly, I'm a little surprised that this wasn't the default...

But I should have checked before merging 🤦‍♂️